client/gutenberg/extensions/map-block/map-theme-picker/style.scss

```diff
@@ -0,0 +1,38 @@
+.component_map-theme-picker__button {
+	border: 1px solid lightgray;
+	border-radius: 100%;
+	width: 70px;
+	height: 70px;
+	margin: 5px;
+	text-indent: -9999px;
+	background-color: lightgray;
+	background-position: center center;
+	background-repeat: no-repeat;
+	background-size: contain;
+	transform: scale(1);
+	transition: transform .2s ease;
+	&:hover {
+		transform: scale(1.1);
+	}
+	&.isSelected {
+		border-color: black;
+	}
+	&.default {
+		background-image: url( 'https://s3.amazonaws.com/atavist-static/map-themes/map-theme-default.jpg' );
```

Comment:
Do we want to host these images in WP.com?

Reply:
Originally I wanted to include these in the block directory, but I ran into this issue: https://github.com/Automattic/wp-calypso/issues/27838. Even if resolved, your question is still quite valid - is it good practice to include assets in source code or should they just go on a CDN. And if CDN is the answer, we should definitely move them from this old Atavist one. Where would I put them on WP.com?

Reply:
I'm not sure if we can use Google map layer image without valid API code in place first. If we can, The image should probably have Google -logo+copyrights at the corner.

Shipping the image with the block feels like the way to go but the SDK might not support it yet. 🤔  WP.com CDN sounds like a sensible alternative meanwhile.

Reply:
Ah, this is a mistake, left in from an ancient Atavist version. I'll remove!

Reply:
Actually, my earlier comment was wrong - these images are indeed meant to be there. These images are used in the circles to demonstrate the different style options:

<img width="278" alt="screen shot 2018-10-19 at 9 54 41 am" src="https://user-images.githubusercontent.com/1477002/47222556-10c11300-d385-11e8-913f-ac70a286afc2.png">
 
The WP.com CDN solution sounds like the right approach, but @simison can you say a little more about what you meant by this?

> I'm not sure if we can use Google map layer image without valid API code in place first. If we can, the image should probably have Google -logo+copyrights at the corner.

Reply:
I meant that typically Google isn't very happy if their map tiles are used without logo/copyright attribution visible somewhere, but not sure if it matters in this case. I'm sure it's buried somewhere in their [terms](https://cloud.google.com/maps-platform/terms/).

Could these images be loaded directly from Google servers via [Maps Static API](https://developers.google.com/maps/documentation/maps-static/intro), using the same API key that user provides?

The theme selection could stay hidden until the API key is provided.

Reply:
Related PR to include image assets (pngs): #28378

Reply:
This question resolved and the CDN question superseded by the work in https://github.com/Automattic/wp-calypso/pull/28378 (thank you @sirreal!) Moved the images into the bundle in https://github.com/Automattic/wp-calypso/pull/27836/commits/ea07f2e853409425716133aed494a53afd9d2107